pmd-core/src/main/java/net/sourceforge/pmd/lang/AbstractCpdLanguageVersionHandler.java

```diff
@@ -1,34 +0,0 @@
-/**
- * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
- */
-
-package net.sourceforge.pmd.lang;
-
-import java.util.Collections;
-import java.util.List;
-
-import net.sourceforge.pmd.lang.ast.AstProcessingStage;
-import net.sourceforge.pmd.lang.rule.RuleViolationFactory;
-
-
-/**
- * Base language version handler for languages that only support CPD.
- *
- * @author Clément Fournier
- * @since 6.10.0
```

Comment:
This class looks like it is from PMD 6, but I don't find it there.... Is this save to delete?

Reply:
In fact the class is from #1426, so has never been on master. I must have messed up the since attribute. 

See https://github.com/pmd/pmd/pull/1426/files#diff-4646a0d1712c24f6adb57dd98c36bc43